editor/debugger/editor_profiler.cpp

```diff
@@ -288,6 +288,14 @@ void EditorProfiler::_update_plot() {
 					column[j * 4 + 1] += Math::fast_ftoi(CLAMP(col.g * 255, 0, 255));
 					column[j * 4 + 2] += Math::fast_ftoi(CLAMP(col.b * 255, 0, 255));
 					column[j * 4 + 3] += 1;
+
+					if (EDSCALE > (1.5 - CMP_EPSILON)) {
+						// Make the line thicker for hiDPI displays.
+						column[j * 4 + 4] += Math::fast_ftoi(CLAMP(col.r * 255, 0, 255));
+						column[j * 4 + 5] += Math::fast_ftoi(CLAMP(col.g * 255, 0, 255));
+						column[j * 4 + 6] += Math::fast_ftoi(CLAMP(col.b * 255, 0, 255));
+						column[j * 4 + 7] += 1;
+					}
```

Comment:
This should have additional check for `j` value. Not sure if this ever happens, but seems like `plot_pos` can point to the last column (`h - 1`), and in this case it will go out of bounds of `columns` (size `h * 4`).